src/calendar/view/eventeditor/CalendarEventEditView.ts

```diff
@@ -270,4 +289,58 @@ export class CalendarEventEditView implements Component<CalendarEventEditViewAtt
 		vnode.attrs.descriptionEditor.setEnabled(model.isFullyWritable())
 		return m(vnode.attrs.descriptionEditor)
 	}
+
+	private showCustomReminderIntervalDialog(onAddAction: (value: number, unit: AlarmIntervalUnit) => void) {
+		let timeReminderValue = 0
+		let timeReminderUnit: AlarmIntervalUnit = AlarmIntervalUnit.MINUTE
+
+		Dialog.showActionDialog({
+			title: () => lang.get("calendarReminderIntervalCustomDialog_title"),
+			allowOkWithReturn: true,
+			child: {
+				view: () =>
+					m(".flex full-width pt-s", [
+						m(TextField, {
+							type: TextFieldType.Number,
+							min: 0,
+							label: "calendarReminderIntervalValue_label",
+							value: timeReminderValue.toString(),
+							oninput: (v) => {
+								const time = Number.parseInt(v)
+								const isEmpty = v === ""
+								if (!Number.isNaN(time) || isEmpty) timeReminderValue = isEmpty ? 0 : time
+							},
+							class: "text flex-half no-appearance", //Removes the up/down arrow from input number. Pressing arrow up/down key still working
+						}),
+						m(TextField, {
```

Comment:
`DropdownSelector` (I think then we also get the dropdown opening when you click the text)